libretroshare/src/pqi/p3netmgr.cc

```diff
@@ -55,6 +55,7 @@ const uint32_t RS_NET_EXT_SETUP =  	0x0004;
 const uint32_t RS_NET_DONE =    	0x0005;
 const uint32_t RS_NET_LOOPBACK =    	0x0006;
 //const uint32_t RS_NET_DOWN =    	0x0007;
+const uint32_t RS_NET_SHUTDOWN =    0x00FF; //Highest value to not restart UPnP nor ExtAddrFinder
```

Comment:
`constexpr` seems more appropriate to me

Reply:
Done